AirCasting/Graph/Graph.swift

```diff
@@ -100,8 +100,9 @@ struct Graph: UIViewRepresentable {
                                                     matching: components,
                                                     matchingPolicy: Calendar.MatchingPolicy.previousTimePreservingSmallerComponents,
                                                     repeatedTimePolicy: Calendar.RepeatedTimePolicy.first,
-                                                    direction: .backward)
-            if let midnightPoint = previous?.timeIntervalSince1970 {
+                                                     direction: .backward)
+            
+            if let midnightPoint = previous?.currentUTCTimeZoneDate.timeIntervalSince1970 {
```

Comment:
are we sure we want a UTC? I'm a bit confused on how the time's working in our app right now - did you consult it with Marysia? :D